src/controllers/userContoller.js

```diff
@@ -0,0 +1,103 @@
+const {
+  getUsers,
+  addUser,
+  getUser,
+  deleteUser,
+  updateUser,
+} = require('../services/users.service');
+
+const getAll = (req, res) => {
+  res.statusCode = 200;
+
+  const users = getUsers();
+
+  res.send(users);
+};
+
+const add = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const rawUser = {
+    name,
+  };
+
+  const user = addUser(rawUser);
```

Comment:
Ensure that `addUser` handles any potential errors or exceptions, such as duplicate entries, and returns an appropriate response.